src/verbs/queue_pair.rs

```diff
@@ -0,0 +1,22 @@
+use rdma_mummy_sys::{self, ibv_qp, ibv_qp_init_attr_ex};
+
+#[derive(Debug, Clone, Copy)]
+pub struct QueuePairInitAttr {
+    _attr_ptr: *mut ibv_qp_init_attr_ex,
+}
+
+impl QueuePairInitAttr {}
+
+pub struct QueuePair {
+    pub(crate) qp_ptr: *mut ibv_qp,
```

Comment:
A question about naming convention: I notice that all the pointers are named `*_ptr`. Is this a preferred naming convention in rust?

Reply:
There is no such convention, just personal preference, I think it would make our internal implementation clearer?

For example, if we have a object of `QueuePair` named `qp`, then we can access the inner pointer with `qp.qp_ptr`, but if we name the pointer with `qp`, then it would be `qp.qp`, looks weired